lib/Runtime/PlatformAgnostic/ChakraICU.h

```diff
@@ -8,7 +8,13 @@
 
 #ifdef HAS_ICU
 #ifdef WINDOWS10_ICU
+// if WINDOWS10_ICU is defined, pretend like we are building for recent Redstone,
+// even if that isn't necessarily true
+#pragma push_macro("NTDDI_VERSION")
```

Comment:
whoa, TIL push_macro -- does this work the same on xplat?

Reply:
This is exclusively for Windows, but no, I don't think it does. Looked to be MSVC-specific